data/filter.go

```diff
@@ -91,3 +100,53 @@ var CorporateInformation = ContentType{
 	Type:            "corporate_information",
 	SubTypes:        []string{"static_foi", "static_page", "static_landing_page", "static_article"},
 }
+
+var errFilterType = errors.New("invalid filter type given")
+
+func setCountZero(categories []Category) []Category {
+	for i, category := range categories {
+		categories[i].Count = 0
+		for j := range category.ContentTypes {
+			categories[i].ContentTypes[j].Count = 0
```

Comment:
Same here what if `.ContentTypes[j]` is nil?

Reply:
Same as above